### DISCUSSION TITLE
last mission broken

### DISCUSSION THREAD

**Discussion by wes1038** [IBAX-io/go-ibax#1747] [Rewards]
hi,

The latest 5000 point retweet task is not working. Even though I retweet, it gives an error when I say finished.

**Comment by CoinEngineer** [comment]
Sistem hata veriyor , aynı sorun bendede oldu.

**Reply by ahmetnur99** [reply]
Neden böyle bir hata veriyor anlamadım

**Comment by Aaltf** [comment]
> 

Yes me and my friends trying aince morning.. but its not working:(
